pkg/network/encoding/format_usm_test.go

```diff
@@ -92,10 +93,43 @@ func TestFormatProtocols(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:       "GnuTLS - unknown protocol",
+			protocol:   network.ProtocolUnknown,
+			staticTags: 1 << 0,
+			want: &model.ProtocolStack{
+				Stack: []model.ProtocolType{
+					model.ProtocolType_protocolTLS,
+					model.ProtocolType_protocolUnknown,
+				},
+			},
+		},
+		{
+			name:       "OpenSSL - unknown protocol",
+			protocol:   network.ProtocolUnknown,
+			staticTags: 1 << 1,
+			want: &model.ProtocolStack{
+				Stack: []model.ProtocolType{
+					model.ProtocolType_protocolTLS,
+					model.ProtocolType_protocolUnknown,
+				},
+			},
+		},
+		{
+			name:       "GoTLS - unknown protocol",
+			protocol:   network.ProtocolUnknown,
+			staticTags: 1 << 2,
+			want: &model.ProtocolStack{
+				Stack: []model.ProtocolType{
+					model.ProtocolType_protocolTLS,
+					model.ProtocolType_protocolUnknown,
+				},
+			},
```

Comment:
1. let's use the consts
2. let's mix the tests, one test with unknown, one with http and the left with db protocol
3. let's add a test for unsupport tag (assume `1 << 10`)